lib/hackney/income/tenancy_prioritiser/tenancy_classification.rb

```diff
@@ -222,7 +222,15 @@ def balance_is_in_arrears_by_amount?(amount)
         end
 
         def balance_with_1_week_grace
-          @criteria.balance - @criteria.weekly_gross_rent
+          @criteria.balance - calculate_grace_amount
+        end
+
+        def calculate_grace_amount
```

Comment:
```suggestion
        def calculated_grace_amount
```
nitpick, since this method also returns the grace_amount